### PR TITLE
Console views test

### PR DESCRIPTION
**Pull request by alishamohanty**
The changes are build on top of the PR #148

### CODE REVIEW

File: test/goose/console_test.clj [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      (is (= (tu/get-called?) false))
+      (console/app-handler {:broker       tu/redis-producer
+                            :app-name     ""
+                            :route-prefix "/goose/console"}
+                           (mock/request :get "/goose/console/"))
+      (is (= (tu/get-called?) true)))))
+
+(deftest homepage-view-test
+  (let [worker (w/start tu/redis-worker-opts)
+        job-opts (assoc tu/redis-client-opts :retry-opts (assoc tu/retry-opts :max-retries 0))
```
Comment (prabhanshuguptagit) [inline test/goose/console_test.clj:28]: Move these inside the deftest.

File: test/goose/console_test.clj [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        _d1 (:id (c/perform-async job-opts `/ 10 0))
+        _ (w/stop worker)
+        _e1 (c/perform-async tu/redis-client-opts `tu/my-fn 1)
```
Comment (prabhanshuguptagit) [inline test/goose/console_test.clj:31]: Consider using full names if needed. `_enqueued-job-1`, `scheduled-job-1` etc. You could move these outside the let and into the test perhaps also so you don't have to name them.

File: src/goose/brokers/redis/console.clj [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+     :periodic  periodic
+     :dead      dead}))
+
+;;Page handler
+(defn home-page [broker {{:keys [app-name]} :client-opts}]
+  (let [view (layout header stats-bar)
+        data (jobs-size (:redis-conn broker))]
+    (response/response (view "Home" (assoc data :app-name app-name)))))
+
+;; Main Handler
```
[the remark this answers is not in the document]
Comment (alishamohanty) [reply src/goose/brokers/redis/console.clj:68]: Addressed this in the previous PR

File: test/goose/console_test.clj (hunk 1 above)
Comment (alishamohanty) [reply test/goose/console_test.clj:28]: I am assuming, you are suggesting to move the worker and job-opts from the deftest block to the testing block, I have moved them.

File: test/goose/console_test.clj (hunk 2 above)
Comment (alishamohanty) [reply test/goose/console_test.clj:31]: I have moved them to the testing block.

**Review by ssrihari — APPROVED** [review]

**Review by prabhanshuguptagit — APPROVED** [review]

**Pull request merged**